transifex/native/tools/migrations/save.py

```diff
@@ -159,3 +168,24 @@ def save_file(self, file_migration):
             file_migration.compile,
             file_type='Original',
         )
+
+
+def create_save_policy(policy_id):
+    """Create the save policy object that corresponds to the given ID.
+
+    :param str policy_id: the ID of the policy to create, as expected
+        in the command parameters
+    :return: a SavePolicy subclass
+    :rtype: SavePolicy
+    """
+    policy_id = policy_id.lower()
+    if policy_id == NoopSavePolicy.name:
+        return NoopSavePolicy()
+    elif policy_id == NewFileSavePolicy.name:
+        return NewFileSavePolicy()
+    elif policy_id == BackupSavePolicy.name:
+        return BackupSavePolicy()
+    elif policy_id == ReplaceSavePolicy.name:
+        return ReplaceSavePolicy()
```

Comment:
Just mentioning, a mapping dictionary may be a leaner way to perform the create_<whatever>_policy.